Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM alpine:latest
+
+# Set the Current Working Directory inside the container
+WORKDIR $GOPATH/src/github.com/esell/deb-simple
+
+# Copy everything from the current directory to the PWD (Present Working Directory) inside the container
+COPY deb-simple /
+
+# This container exposes port 8080 to the outside world
```

Comment:
One very minor nitpick @alexanderturner , any chance you can update this to match the port being exposed for clarity sake?

Reply:
Done :) Sorry about that!